example/development-profile.gradle

```diff
@@ -0,0 +1,50 @@
+apply plugin: "com.android.application"
+
+android {
+    compileSdkVersion 27
+
+    dataBinding.enabled true
+
+    defaultConfig {
+        applicationId "org.aerogear.mobile.example"
+
+        minSdkVersion project.ext.minSdkVersion
+        targetSdkVersion project.ext.targetSdkVersion
+
+        manifestPlaceholders = ["appAuthRedirectScheme": "org.aerogear.mobile.example"]
+
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+}
+
+dependencies {
+    implementation project(path: ":core")
+    implementation project(path: ":auth")
+    implementation project(path: ":push")
```

Comment:
if the app is move to a different repo, how the references here would look like?

Reply:
The app should use the `production-profile.gradle` file as standard as it references a release version of the SDK, it is only for development would we use the `development-profile.gradle` once it has been cloned into the SDK.

Reply:
right, gotcha. So there should be a script to copy the app from its own repo to the SDK's repo (or symlink). Is that what you have in mind?

Reply:
Initially no, I was thinking of manually cloning the showcase into the SDK for testing locally, but it would make a lot more sense that if we are following this approach to write a script to clone and copy the showcase for us, and set the gradle property `profile` to `development`

Reply:
I would say that the script should be there to make the setup for development as easy as possible.

Reply:
Cool, Ill get on that now, and will sync up with @TommyJ1994 later on testing it with the new showcase app